java/INSTALL

```diff
@@ -103,16 +108,16 @@ general overview of OpenDDS.
      use paths with no spaces (true on Unix too but spaces are rarely seen).
      The top level of the OpenDDS distribution is a directory that contains
      the subdirectory named dds.
-
+     
   3. Configure and build OpenDDS 
 
      Build as in $DDS_ROOT/install, but add --java option
 
-     Unix =>
-       ./configure --java=<full path to the JDK>
-     Windows => 
-       configure --java=<full path to the JDK>
-
+     Unix => cd $DDS_ROOT
+             ./configure --java
+             make       
+     Windows => configure --java=<full path to the JDK>
```

Comment:
The problem is that JAVA_HOME is not set automatically during JDK installation. If 
it is not set, there is no logic in configure script to detect where is JDK based
on the location of javac currently. 

Huang-Ming


> On Sep 13, 2016, at 11:23 AM, Adam Mitz <notifications@github.com> wrote:
> 
> In java/INSTALL <https://github.com/objectcomputing/OpenDDS/pull/302#discussion_r78594215>:
> 
> >    3. Configure and build OpenDDS 
> >  
> >       Build as in $DDS_ROOT/install, but add --java option
> >  
> > -     Unix =>
> > -       ./configure --java=<full path to the JDK>
> > -     Windows => 
> > -       configure --java=<full path to the JDK>
> > -
> > +     Unix => cd $DDS_ROOT
> > +             ./configure --java
> > +             make       
> > +     Windows => configure --java=<full path to the JDK>
> I don't know why this requires platform-specific behavior. Can't we look for javac on the PATH (or presence of JAVA_HOME) just as easily on Windows?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/objectcomputing/OpenDDS/pull/302/files/15a960d61ac519cc453485d4640d2aa233ce0494#r78594215>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ADwEIx_UkUH47A92-UFICwtdhqgvPA-lks5qps3_gaJpZM4J73YU>.
>